### PR TITLE
Overhauling Contribution notes

### PR DESCRIPTION
**Pull request by martinheidegger**
Added things that I learned during cleaning the issues. I think this should be useful.

### CODE REVIEW

**Comment by maxogden** [comment]
WOOT

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,21 +1,87 @@
...
 
 Don't email workshop authors directly, use the public discussion forum (the Issues) for all communication, that way your question and any answers you get can benefit everyone else.
 
-## 3 - Be patient
+### 3 - Be patient
 
 It may take a day or two for people to respond here. Please don't send multiple messages in a row, it won't help your question get answered any faster.
 
-# Suggestions
+### 4 - Non-English is okay!
```
Comment (sudodoki) [inline CONTRIBUTING.md]: Might add a note for trying local chapter first.

**Comment by sudodoki** [comment]
:+1: great edits

**Comment by martinheidegger** [comment]
Added some more sections and a TOC, now it should be fine?!

File: CONTRIBUTING.md (hunk 1 above)
Comment (martinheidegger) [inline CONTRIBUTING.md]: Oops, i forgot to add this. Also - while thinking on it - I am just really happy when a non-english speaker reads is able to read this section at all.

**Pull request merged**